recipes/phytop/src/small_tools.py

```
@@ -0,0 +1,358 @@
+		f.close()
+	return _run_time
+
+class pypsl:
+	def __init__(self, inPsl):
+		self.input = inPsl
+	def read(self):
+		title = ''
+		for line in open(self.input, 'r'):
+			if re.compile(r'\d+').match(line):
+				self.match = int(temp[0])
+				self.mis_match = int(temp[1])
```

Ruff

Undefined name `temp`

(F821)